src/app/pages/MediaAssetPage/fixtureData.json

```
@@ -1860,7 +1860,7 @@
   },
   "promo": {
     "headlines": {
-      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >",
+      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
```

```suggestion
      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh, bye bye Khoa & < >",
```

Just a small suggestion :trollface:

---

😂 and also 😭

---

:laughing-but-crying-face:

---

😆 Cheers @ryanmccombe